code/utils/id.h

```diff
@@ -48,7 +48,7 @@ class ID
 
 	inline bool isValid() const { return m_val != default_value; }
 
-private:
+protected:
```

Comment:
Doesn't seem to be a relevant change?

Reply:
It's necessary because otherwise `m_val` would not be visible in the `UI_TIMESTAMP` subclass.